doc/field_guide__eclipsing.md

```diff
@@ -0,0 +1,32 @@
+## Eclipsing variables (ecl)
+
+Eclipsing variables are binary stars systems in which the orbital plane lies nearly in the line of sight of the observer. The binary components pass in front of one another from observer's perspective and so their combined brightness decreases periodically.
+

```

Comment:
Maybe say something here briefly about the eclipses can be different shapes and depths, which you get to below?